dev/src/bulk-writer.ts

```diff
@@ -728,3 +747,76 @@ export class BulkWriter {
     this.maxBatchSize = size;
   }
 }
+
+/**
+ * Validates the use of 'value' as BulkWriterOptions.
+ *
+ * @private
+ * @param arg The argument name or argument index (for varargs methods).
```

Comment:
You can probably be more specific since this is only used for BulkWriterOptions.

Reply:
removed used of `arg` as a parameter.